chain-api/src/types/DexDtos.ts

```diff
@@ -675,9 +670,19 @@ export class CollectProtocolFeesResDto extends ChainCallDTO {
 export class SetProtocolFeeResDto extends ChainCallDTO {
   @IsNumber()
   @IsNotEmpty()
```

Comment:
```suggestion
  @IsNumber()

```

If this number should not be negative, considering including `@IsPositive()`.

Reply:
Since these inputs are expected to accept a wide range of numbers, we have opted not to include additional decorators. Instead, we have simply removed the `@IsNotEmpty` decorator.